.gitignore

```diff
@@ -1,6 +1,7 @@
 node_modules/
 out/
 test*.js
+.idea/
```

Comment:
Ignores specific to your environment should be defined only on your local machine. You can add it to [`.git/info/excludes`](https://luisdalmolin.dev/blog/ignoring-files-in-git-without-gitignore/#:~:text=To%20ignore%20untracked%20files%2C%20you,tracking%20any%20(untracked)%20file.) in your local clone of this repository.